test/python/gcn.py

```diff
@@ -4,9 +4,10 @@
 
 from mpact.mpactbackend import mpact_jit, mpact_jit_compile, mpact_jit_run
 
-from mpact.models.gcn import graphconv44
+from mpact.models.gcn import graphconv44, gcn4164
 
 net = graphconv44()
+net.eval()  # switch to inference
```

Comment:
nit: period in the end? Same as the following comments.

Reply:
to nit back: inline comments (at the end) don't get the period
(following google style)

Reply:
I stand corrected, C++ uses that, but not Python.